packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -510,6 +510,14 @@ export const ChatScreen = () => {
     flatListInnerHeight.current = event.nativeEvent.layout.height
   }, [])
 
+  // Scroll to the bottom if the user sends a message
+  const handleMessageSent = useCallback(() => {
+    // Set a timeout to ensure the full render happened and we got the final height
+    setTimeout(() => {
+      flatListRef.current?.scrollToOffset({ offset: 0, animated: true })
+    }, 0)
```

Comment:
nit: According to the [docs](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#:~:text=a%20security%20risk.-,delay,-Optional), omitting delay causes it to default to `0` and has 'execute immediately' semantics.

Reply:
Just confirming you tested it both within and outside of the range we pass to `maintainVisibleContent`. That changes the default behavior and I wonder if there is any strange behavior as a result.

Reply:
Execute next event cycle is what I want I think - it just needs to happen after a single render cycle. Didn't test too heavily with the `maintainVisibleContent` stuff, but I think since this is guaranteed to execute last it should have no oddities